src/content/release/breaking-changes/uiscenedelegate.md

```diff
@@ -47,9 +47,28 @@ UIKit won't call AppDelegate methods related to UI state.
 
 ## Migration guide for Flutter apps
 
-The Flutter CLI will automatically migrate your app when you run `flutter run`
-or `flutter build ios` if your AppDelegate has not been customized. Otherwise,
-you must migrate manually.
+### Auto-Migrate (Experimental)
+
+The Flutter CLI can automatically migrate your app if your AppDelegate has not
+been customized.
+
+1. Enable UIScene Migration Feature
+
+```console
+flutter config --enable-uiscene-migration
+```
+
+2. Build or run your app
+
+```console
+flutter run
+or
+flutter build ios
+```
+
+If the migration succeeds, you will see a log that says "Finished migration to
+UIScene lifecycle". Otherwise, it will warn you to migrate manually using the
+below instructions. If the migration succeeds, no further action is required!
```

Comment:
```suggestion
UIScene lifecycle". Otherwise, it warns you to migrate manually using the
included instructions. If the migration succeeds, no further action is required!
```